flink-streaming-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/internals/KafkaTopicPartition.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
The file has inconsistent control flow style, some of the `if` have a space before opening parenthesis the `for` loops don't have a space.